.github/workflows/python-package.yml

```diff
@@ -62,13 +62,57 @@ jobs:
     - name: >
         Run tox for
         "${{ matrix.python-version }}-unit"
-      timeout-minutes: 20
+      timeout-minutes: 25
       run: |
         tox --verbose --verbose
 
     - uses: codecov/codecov-action@v2.1.0
       with:
         flags: unittests # optional
-        fail_ci_if_error: true # optional (default = false)
+#        fail_ci_if_error: true # optional (default = false)
         verbose: true # optional (default = false)
 
+  Integration:
+      needs:
+      - Unit
+      if: needs.Unit.result == 'success'
+      timeout-minutes: 240
+
+      runs-on: ubuntu-20.04
+      strategy:
+          fail-fast: false
+          matrix:
+              python-version: ['3.7', '3.8', '3.9']
```

Comment:
We support 3.7 - 3.10. I assume you just wanted a shorter build, right?

Reply:
Added to be sure that everything is working. I also added if statement to be sure that Integration is run only, if Unit tests were successful